python/lsst/meas/modelfit/display/cModelDisplay.py

```diff
@@ -0,0 +1,264 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+import lsst.afw.image as afwImage
+import lsst.meas.modelfit as measMod
+import lsst.shapelet as shapelet
+
+
+def displayReconstruedCmodel(exposure, record, cModelConfig, display="mpl"):
+    """ Display an image, the Cmodel, and residuals
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`
+        Exposure object that contains the source which was  modeled
+    record : `lsst.afw.table.SourceRecord`
+        Record object which contains the measurements made on the source
+    cModelConfig : `lsst.meas.modelfit.CModel(SingleFrame/Forced)Config`
```

Comment:
Since there are no other configuration arguments, just call it `config`?

(just because we have so many other keyword arguments that are just called `config`; it's almost a convention)